front/app/containers/Admin/messaging/texting/CreatedSMSCampaignForm/index.tsx

```diff
@@ -154,73 +153,66 @@ const CreatedSMSCampaignForm = (props: WithRouterProps) => {
-                </span>
-              )}
-              {!isDraft && <span>Sent to:</span>}
-            </Label>
-            <TextArea
-              rows={8}
-              maxRows={8}
-              value={inputPhoneNumbers}
-              disabled={!isDraft}
-              onChange={handleInputPhoneNumbersChange}
-            />
-          </SectionField>
-          <SectionField>
-            <Label>
-              Message <IconTooltip content="Help goes here" />
-            </Label>
-            <TextArea
-              rows={8}
-              maxRows={8}
-              value={inputMessage}
-              disabled={!isDraft}
-              onChange={handleInputMessageChange}
-            />
+      <TextingHeader
+        headerMessage={getTitleMessage(status)}
+        onClickGoBack={clHistory.goBack}
+      />
+      <>
+        <Box display="inline-block" marginRight="12px" marginBottom="24px">
+          <FormattedStatusLabel campaignStatus={status} />
+        </Box>
+        <span>{getAdditionalInfoByStatus(campaign)}</span>
+      </>
+      <StyledForm onSubmit={handleOnSubmit}>
+        <Box marginBottom="20px">
+          <Label>
             {isDraft && (
-              <RemainingCharacters
-                remainingChars={remainingChars}
-                overCharacterLimit={overCharacterLimit}
-              />
+              <span>
+                Enter a list of phone numbers. Separate each number by a comma
+                and include the international dialing code (eg. +1).
+              </span>
             )}
-          </SectionField>
-
+            {!isDraft && <span>Sent to:</span>}
+          </Label>
+          <TextArea
+            rows={8}
+            maxRows={8}
+            value={inputPhoneNumbers}
+            disabled={!isDraft}
+            onChange={handleInputPhoneNumbersChange}
+          />
+        </Box>
+        <Box marginBottom="30px">
+          <Label>
+            Message <IconTooltip content="Help goes here" />
+          </Label>
+          <TextArea
+            rows={8}
+            maxRows={8}
+            value={inputMessage}
+            disabled={!isDraft}
+            onChange={handleInputMessageChange}
+          />
           {isDraft && (
-            <SectionField>
-              <Box maxWidth="250px">
-                <Button
-                  buttonStyle="primary"
-                  size="2"
-                  type="submit"
-                  text={'Update and Preview SMS'}
-                  onClick={handleOnSubmit}
-                  disabled={overCharacterLimit}
-                />
-              </Box>
-            </SectionField>
+            <RemainingCharacters
+              remainingChars={remainingChars}
+              overCharacterLimit={overCharacterLimit}
+            />
           )}
-        </StyledForm>
-      </Section>
+        </Box>
+
+        {isDraft && (
+          <Box maxWidth="250px">
```

Comment:
You could set width on the button? Or is this the preferred method?

Reply:
Changing this to a flex display actually, a fixed width or max-width might cause issues with translations